catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/forms/builder/XmlModelBuilderTest.java

```diff
@@ -136,24 +138,6 @@ public void testComplexEmbeddedLogicNodes() throws Exception {
         .verifyTerminalNode("/Filter/And/Or/PropertyIsLessThanOrEqualTo", "width", "20");
```

Comment:
✏️ I want to get rid of all this xpath. I made a DSL for verifying JAXB structures and should probably use that instead. Debating if it should be done on this PR.

Reply:
Ended up keeping the XPath for now since ideally we move to Geotools for our XML binding. In that case, there won't be any JAXB objects to verify, and the DSL should be updated to use XPath in the background instead.